packages/base/src/UI5Element.js

```diff
@@ -497,6 +502,10 @@ class UI5Element extends HTMLElement {
 			// console.log("INVAL", this, ...arguments);
 			RenderScheduler.renderDeferred(this);
 		}
+
+		if (this.shouldInvalidateParent) {
```

Comment:
I'm not 100% sure yet (we'd have to experiment) but I think we should move this `if` immediately after `RenderScheduler.renderDeferred(this)` so that we don't invalidate the parent unless the child is effectively invalidated (i.e. scheduled for rendering)